build/main.yml

```diff
@@ -185,19 +178,12 @@ jobs:
     - task: DockerInstaller@0
       inputs:
         dockerVersion: '17.09.0-ce'
-    - task: Bash@3
-      displayName: Install dependencies
-      inputs:
-        targetType: 'inline'
-        script: |
-          # Write your commands here
```

Comment:
Not required, as we're using a docker container
this is a result of copy n paste